README.md

```diff
@@ -131,11 +131,13 @@ Within the Disk Management window, shrink your Windows Volume to create an empty
 
 To check if your computer supports *Device Encryption*, open **Settings > Privacy & Security**. If your computer support that feature, you should see a tab called *Device Encryption*. By default, Windows 11 should turn the feature on.
 
-**NOTE: the recovery key is linked to the microsoft account associated with the user. You MUST therefore have linked Windows 11 to your Microsoft Account !**
+**NOTE: when the Device Encryption is over, you should check and save the recovery key somewhere safe:**
+* **If you linked your user account to your Microsoft Account: the recovery key can be obtained through [this Microsoft link](https://go.redirectingat.com/?id=111346X1569483&url=https://account.microsoft.com/devices/recoverykey&xcust=2-1-624599-1-0-0&sref=https://www.pcworld.com/article/624599/how-to-find-the-recovery-key-for-your-encrypted-windows-11-pc.html)**
+* **Otherwise, go to Settings > Privacy & Security >  and click on "Get recovery key"**
```

Comment:
WIP: @Malian could you show me again the procedure for getting the recovery key after turning on "Device Encryption" ? I can't remember exactily. Tks